### PR TITLE
feature/bills-tab-home

### LINKED ISSUE #239: Feature Request: view recurring transactions/bills on Home Screen
- add new view "recurring transactions/bills" on Home Screen tabs

### PR DESCRIPTION
**Pull request by daanvanberkel**
## Add tab for bills on homescreen

**Summary**
I added an extra tab on the home screen to view the bills for the selected period. Including a progress bar to show how much still must be paid for the bills.

### CODE REVIEW

**Comment by victorbalssa** [comment]
Thank you @daanvanberkel! 
I will review it 👍🏻

File: src/components/Screens/HomeScreen.tsx [hunk 1]
```diff
@@ -244,6 +245,72 @@ function InsightBudgets() {
...
+          <AText
+            fontSize={15}
+            numberOfLines={1}
+            color={colors.brandSuccess}
+            style={{ textAlign: 'center' }}
+            bold
+          >
+            {`${((totalPaid / total) * 100).toFixed(0)}%`}
+          </AText>
+        </AStack>
```
Comment (victorbalssa) [inline src/components/Screens/HomeScreen.tsx]: Can you wrap this stack to conditionally display it if only `total` !== 0?

For older periods where I have bills but nothing is expected total return 0, displaying `NaN` in the "bills summary".

File: src/components/Screens/HomeScreen.tsx [hunk 2]
```diff
@@ -244,6 +245,72 @@ function InsightBudgets() {
...
+      </AStack>
+
+      <AStack mx={15} justifyContent="flex-start">
+        <AProgressBar
+          color={colors.green}
+          value={(totalPaid / total) * 100}
+        />
+      </AStack>
```
Comment (victorbalssa) [inline src/components/Screens/HomeScreen.tsx]: (same comment) Can you wrap this stack to conditionally display it if only `total` !== 0?

For older periods where I have bills but nothing is expected total return 0, displaying `NaN` in the "bills summary".

**Review by victorbalssa — COMMENTED** [review]
Looks perfect! 👀 
 
 Thank you for following the current structure and custom view components 👍🏻 

Just 2 comments to fix edge cases, and I can send it to OTA.

**Comment by daanvanberkel** [comment]
Hi @victorbalssa, thanks for the review! I fixed your comments 😄

**Review by victorbalssa — APPROVED** [review]
Thank you @daanvanberkel, approved.

**Pull request merged**